crates/common/batcher/Cargo.toml

```diff
@@ -2,7 +2,8 @@
 name = "batcher"
 version = "0.5.1"
 authors = ["thin-edge.io team <info@thin-edge.io>"]
-edition = "2018"
+edition = "2021"
+rust-version = "1.58"
```

Comment:
Do we really need `1.58` as minimum `rust-version` ?

* For now we use none of these new features - from yesterday ;-).
* That said, I agree that using "Disjoint capture in closures" and "Captured identifiers in format strings" will be really helpful.